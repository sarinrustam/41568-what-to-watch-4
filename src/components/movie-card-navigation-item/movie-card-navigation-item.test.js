import React from "react";
import renderer from "react-test-renderer";

import MovieCardNavigationItem from './movie-card-navigation-item.jsx';


describe(`MovieCardDescriptionItem render`, () => {
  it(`MovieCardDescriptionItem render correctly`, () => {
    const handlerSetActive = jest.fn();
    const tree = renderer.create(
        <MovieCardNavigationItem
          filter={`Default`}
          activeFilter={`Overview`}
          setActiveFilter={handlerSetActive}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
