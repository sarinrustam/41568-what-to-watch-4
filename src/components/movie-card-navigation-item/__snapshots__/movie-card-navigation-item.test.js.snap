// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardDescriptionItem render MovieCardDescriptionItem render correctly 1`] = `
<li
  className="movie-nav__item"
  onClick={[Function]}
>
  <a
    className="movie-nav__link"
    href="#"
  >
    Default
  </a>
</li>
`;
